Lua/Lua.sublime-syntax

```diff
@@ -303,6 +304,20 @@ contexts:
 
     - include: else-pop
 
+  variable-modifier:
+    - match: \<(?=\b(?:const|close)\b)
+      scope: punctuation.definition.modifier.begin.lua
+      push:
+        - variable-modifier-body
+
+  variable-modifier-body:
+    - meta_scope: meta.modifier.lua
+    - match: \>
```

Comment:
I'd prefer to YAML-escape this rather than regxp-escape, i.e. use `'>'`, but @deathaxe disagrees.

Reply:
Anyway, I prefered to make you happy with regards to HTML. :)

Reply:
I just tried using `('>')` instead of `\>` but that doesn't seem to work.

Reply:
There shouldn't be parens around the single quotes